ci-operator/step-registry/telco5g/gather-pao/telco5g-gather-pao-commands.sh

```diff
@@ -23,16 +23,33 @@ fi
 
 echo "************ telco5g gather-pao commands ************"
 
-pao_mg_tag="4.11" # pao must-gather does not have 'latest' tag - setting 4.11 as a workaround for now.
-PULL_BASE_REF=${PULL_BASE_REF:-"master"}
-if [ ${PULL_BASE_REF} != "master" ]
-then
-        pao_mg_tag=${PULL_BASE_REF##release-}
+if [[ -n "${E2E_TESTS_CONFIG:-}" ]]; then
+    readarray -t config <<< "${E2E_TESTS_CONFIG}"
+    for var in "${config[@]}"; do
+        if [[ ! -z "${var}" ]]; then
+            if [[ "${var}" == *"CNF_BRANCH"* ]]; then
+                CNF_BRANCH="$(echo "${var}" | cut -d'=' -f2)"
+            fi
+        fi
+    done
+fi
+
+echo "Running for CNF_BRANCH=${CNF_BRANCH}"
+if [[ "$CNF_BRANCH" == *"4.11"* ]]; then
+    pao_mg_tag="4.11"
+fi
+if [[ "$CNF_BRANCH" == *"4.12"* ]] || [[ "$CNF_BRANCH" == *"master"* ]]; then
+    pao_mg_tag="4.12"
+fi
+if [[ "$CNF_BRANCH" == *"4.13"* ]]; then
+    pao_mg_tag="4.12"
```

Comment:
y are we using 4.12 mg for 4.13 content? what happend to pao mg 4.13?
is that just a temp solution?

Reply:
Nothing is ready for 4.12, even prow part. So we use 4.12 for now. Will be updated when 4.13 is out.

Reply:
Actually prow 4.13 started to work right now. But we'll enable it later.